react-native-libraries.json

```
@@ -9321,5 +9321,12 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/dotnet/aspnetcore/tree/main/src/SignalR",
```

Hello @fiznool,, thank you for adding the new library to the directory! 👍 

```suggestion
    "githubUrl": "https://github.com/dotnet/aspnetcore/tree/main/src/SignalR/clients/ts/signalr",
```

I have one suggestion, GitHub URL need to point to the directory where `package.json` file for wanted package the resides, otherwise the metadata extraction will fail and package won't be visible on the website.